spk/rutorrent/src/service-setup.sh

```diff
@@ -118,8 +118,7 @@ service_postinst ()
         fi
     fi
 
-
-    #If python3 is available setup a virtual environment with cloudscraper
+    # If python3 is available setup a virtual environment with cloudscraper
     if [ -f "${PYTHON_DIR}/bin/python3" ]; then
```

Comment:
My guess is, we could remove this condition and execute the then part whatever happens, since we're relying on python3 to be present.

Reply:
good point, done.